src/org/camunda/latera/bss/connectors/HOMS.groovy

```diff
@@ -167,4 +168,20 @@ class HOMS {
   List attachFile(Map file, String prefix = '', Boolean save = true) {
     return attachFiles([file], prefix, save)
   }
-}
\ No newline at end of file
+
+  void sendTaskEvent(String taskId, String eventName) {
+    LinkedHashMap body = [
+      event_name: eventName
+    ]
+
+    logger.info("/ Sending event ${eventName} for task ${taskId} ...")
+
+    this.http.sendRequest(
+      'put',
+      path: "events/tasks/${taskId}",
```

Comment:
This path is discussable. I thought that we can have not only task events but whole-process events as well so I made extra level for type.